app/build.gradle

```diff
@@ -83,6 +83,10 @@ android {
         testInstrumentationRunnerArgument "TEST_SERVER_URL", "${NC_TEST_SERVER_BASEURL}"
         testInstrumentationRunnerArgument "TEST_SERVER_USERNAME", "${NC_TEST_SERVER_USERNAME}"
         testInstrumentationRunnerArgument "TEST_SERVER_PASSWORD", "${NC_TEST_SERVER_PASSWORD}"
+
+        def localBroadcastPermission = "com.nextcloud.talk.permission.PRIVATE_BROADCAST"
+        manifestPlaceholders.broadcastPermission = localBroadcastPermission
+        buildConfigField "String", "PERMISSION_LOCAL_BROADCAST", "\"${localBroadcastPermission}\""
```

Comment:
If anyone knows a better way to have the constant accessible both in the manifest and in the code, please tell me, I don't really like this.